metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -349,8 +349,8 @@ def get_tables_for_dataset(
                 )
                 if "last_altered" in table
                 else None,
-                size_in_bytes=table.bytes if "bytes" in table else None,
-                rows_count=table.row_count if "row_count" in table else None,
+                size_in_bytes=table.bytes if "bytes" in table.keys() else None,
```

Comment:
Can table be None?

Reply:
table can't be null as this is return bigquery Row objects from the RowIterator